controllers/templates.js

```diff
@@ -28,9 +31,8 @@ templateController.add = (req, res) => {
   if(!valid) return res.json( { errorMessage : ec.templates.INVALID_TEMPLATE } );
   for(let i = 0; i < template.events.length; i++) {
     const e = template.events[i];
-    if(!util.time.isValid(e.startTime)) return res.json( { errorMessage : ec.events.INVALID_TIME_FORMAT } );
-    if(!util.time.isValid(e.endTime)) return res.json( { errorMessage : ec.events.INVALID_TIME_FORMAT } );
-    if(!util.time.isBeforeTime(e.startTime, e.endTime)) return res.json( { errorMessage : ec.events.INVALID_TIME_VALUES } );
+    const error = validatebaseEvent(e);
+    if(error != "") res.json({ errorMessage: error });
```

Comment:
Suggestion: instead of using check `error != ""`,
we can return an object from validation function similar to `{"valid": true, "errors": []}`